spec/cucumber/formatter/ansicolor_spec.rb

```diff
@@ -36,7 +36,7 @@ module Formatter
 
       context 'with custom color scheme' do
         before do
-          apply_custom_colors('passed=red,bold')
+          ANSIColor.apply_custom_colors('passed=red,bold')
```

Comment:
whilst I don't agree with it, can't we just do self.apply..... here as we've got included module in our block.

Ideally this should all be ran in an isolated non-leaking block, but that might be a lot of work for a tiny bugfix PR

Reply:
I tried that and it didn't work: Replacing the explicit `ANSIColor` with `self` (or `self.class` for that matter) still causes a `NoMethodError`. For the sake of completeness, I also tried to `extend ANSIColor` in the tests, but that didn't help either.

IIRC, methods defined in a `Module` become instance methods when the module i included, but module methods do not become class methods, when including the module.

In fact, I'd much rather solve this with a block, as @luke-hill suggests. (And leave this PR as is.)

Reply:
Right now, I would suggest to merge it as-is as it is working fine and it is actually fixing an issue.

We know that we have room for improvements related to ANSIColors. We can do that as part of another PR

@luke-hill what do you think?

Reply:
The problem here is when you include the module those become included as instance methods on the class, but what we're doing here is running inside an rspec context - which is explicitly frowned upon. Ideally we should run in an isolated context. 

Also calling Module.method_name isn't a good practice as this doesn't work in reality, only when they're as either module functions or modified on the eigenclass of the module.

For now if we want to merge AS-IS and it fixes something go for it.

Reply:
I'd like to note that I don't understand why exactly calling `self.class.apply_custom_colors` doesn't work in the RSpec code, since `Cucumber::Formatter::ANSIColor` is in the list of ancestors at the point the `before` is running.